core/networks/42.json

```diff
@@ -70,5 +70,93 @@
   {
     "contractName": "LeveragedReturnCalculator",
     "address": "0xad8fD1f418FB860A383c9D4647880af7f043Ef39"
+  },
```

Comment:
You probably wanna move the old deployment (all the contracts listed above this line) just for cleanliness.